### PR TITLE
Fix deletion message in edit product page

### LINKED ISSUE #20632: BO - Products page - Incorrect wording when we try to delete a related product, a product feature...
#### Describe the bug

In the BO => Catalog => Products page => Product details page => when we try to delete a related product, a product feature, a product brand, a product image, a product combination and bulk delete product combinations, a customization field, a virtual product associated file, a product in a pack, a specific price we have an incorrect wording

image

#### Expected behavior

We should have this message "Are you sure you want to delete this item?" in **Admin.Notifications.Warning** (updated by @LouiseBonnard on August 25th)

#### Steps to Reproduce

1. Go to BO => Catalog => Products page
2. Edit any product
3. Scroll down to Related product section
4. add a related product, a product feature, a product brand, a product image, a product combination, a customization field, a virtual product associated file, a product in a pack, a specific price
5. try to remove them (single and bulk for combinations)
6. pop up us is displayed to confirm the deletion
7. see error

**Screenshots**

https://drive.google.com/file/d/1UfHPy6_CHhN3mXO8meh1KJOZUBUJX9SC/view?usp=sharing

**For QA to check, you must try to delete:**
- [ ] A related product
- [ ] A feature
- [ ] A brand
- [ ] An image
- [ ] A generated combination
- [ ] Combinations in bulk
- [ ] A customization field
- [ ] An associated file to a virtual product
- [ ] A product in a pack
- [ ] A specific price
- [ ] A redirection to a category (in SEO tab)
- [ ] A redirection to a product (in SEO tab)
- [ ] The product currently edited

#### Additional information

* PrestaShop version: 1.7.5.1, 1.7.6.7, 1.7.7.x
* PHP version: 7.2

### PR DESCRIPTION
**Pull request by marionf**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Fix deletion message in edit product page
| Type?         | bug fix
| Category?     | BO 
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #20632
| How to test?  | See the checklist in the issue

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21239)

### CODE REVIEW

File: admin-dev/themes/new-theme/js/product-page/product-search-autocomplete.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ export default function () {
     $(document).on('click', formId, (e) => {
       e.preventDefault();
 
-      window.modalConfirmation.create(window.translate_javascripts['Are you sure to delete this?'], null, {
+      window.modalConfirmation.create(window.translate_javascripts['Are you sure you want to delete this item?'], null, {
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/js/product-page/product-search-autocomplete.js]:
```suggestion
      window.modalConfirmation.create(
        window.translate_javascripts['Are you sure you want to delete this item?'],
        null,
        {
```

File: admin-dev/themes/new-theme/js/product-page/product-bulk-combinations.js [hunk 2]
```diff
@@ -106,7 +106,7 @@ export default function () {
         combinationsIds.push(combination.domId);
       });
 
-      window.modalConfirmation.create(window.translate_javascripts['Are you sure to delete this?'], null, {
+      window.modalConfirmation.create(window.translate_javascripts['Are you sure you want to delete the selected item(s)?'], null, {
```
Comment (Progi1984) [inline admin-dev/themes/new-theme/js/product-page/product-bulk-combinations.js]:
```suggestion
      window.modalConfirmation.create(
        window.translate_javascripts['Are you sure you want to delete the selected item(s)?'],
        null,
        {
```

File: src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/product.html.twig [hunk 3]
```diff
@@ -346,8 +346,8 @@
 "files recommandations": "Recommended size 800 x 800px for default theme."|trans({}, 'Admin.Catalog.Feature'),
 "files recommandations2": "JPG, GIF or PNG format."|trans({}, 'Admin.Catalog.Feature'),
 "Cover": "Cover"|trans({}, 'Admin.Catalog.Feature'),
-"Are you sure to delete this?": "Are you sure to delete this?"|trans({}, 'Admin.Notifications.Warning'),
-"This will delete the specific price. Do you wish to proceed?": "This will delete the specific price. Do you wish to proceed?"|trans({}, 'Admin.Catalog.Notification'),
+"Are you sure you want to delete this item?": "Are you sure you want to delete this item?"|trans({}, 'Admin.Notifications.Warning'),
+"Are you sure you want to delete this item?": "Are you sure you want to delete this item?"|trans({}, 'Admin.Catalog.Notification'),
```
Comment (Progi1984) [inline src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/product.html.twig]:
```suggestion
"Are you sure you want to delete this item?": "Are you sure you want to delete this item?"|trans({}, 'Admin.Notifications.Warning'),
"Are you sure you want to delete this item?": "Are you sure you want to delete this item?"|trans({}, 'Admin.Catalog.Notification'),
```
Duplicate key ? Where is the good catalog ?

Comment (marionf) [reply src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/product.html.twig]: According to @LouiseBonnard [comment](https://github.com/PrestaShop/PrestaShop/issues/20632#issuecomment-679230034) I think it's Admin.Notifications.Warning, maybe the other line with Admin.Catalog.Notification can be deleted ?

Comment (Progi1984) [reply src/PrestaShopBundle/Resources/views/Admin/Product/ProductPage/product.html.twig]: Affirmative ;)

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by ngodefroy** [comment]
Hello @marionf 
Checked delete conf messages:

-  A related product
-  A feature
-  A brand
-  An image
-  A generated combination
-  Combinations in bulk
-  A customization field
-  An associated file to a virtual product
-  A product in a pack
-  A specific price
-  A redirection to a category (in SEO tab)
-  A redirection to a product (in SEO tab)
-  The product currently edited

==>>>TEST OK

Thank you

**Comment by Progi1984** [comment]
Thanks @marionf

**Pull request merged**